.github/workflows/publish-docker-image.yaml

```diff
@@ -0,0 +1,45 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches: main
+
+jobs:
+  push_to_registries:
+    name: Push Docker image to multiple registries
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      
+      - name: Log in to the Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            eoscostarica506/eosio-local
```

Comment:
should image name be `eos-local` not `eosio-local`?

Reply:
sorry, my bad, eos-local is righ

Reply:
Fixed here: https://github.com/eoscostarica/eos-local/pull/88/files#diff-ac9bda107c3925408bf0ffb31e643a4d1e8b9953abc242d8fdd4b32b0a68263aR36